src/Shouldly/ShouldlyExtensionMethods/ShouldBe/StringNullOrEmptyTestExtensions.cs

```diff
@@ -1,43 +1,44 @@
 ﻿using System;
 using JetBrains.Annotations;
+using NotNullAttribute = System.Diagnostics.CodeAnalysis.NotNullAttribute;
 
 namespace Shouldly
 {
     public static partial class ShouldBeStringTestExtensions
     {
         [ContractAnnotation("actual:notnull => halt")]
```

Comment:
Side note: I think these JetBrains annotations might actually be incorrect. If actual is `""`, a non-null value, the method does not halt.